Active_Record/app/models/booking_gears.rb

```diff
@@ -0,0 +1,5 @@
+class BookingGear < ActiveRecord::Base
```

Comment:
Convension over configuration. This model should be named `BookingsGear`

Reply:
does migration also get the wrong name?

Reply:
done